### ISSUE TITLE
Charging cable connected showing wrong data

### ISSUE THREAD

**Issue by flopp999** [lendy007/skodaconnect#16]
response from myskoda
`Request for "https://api.connect.skoda-auto.cz/api/v1/charging/TMBJBxxxxxxxxxxxx/status" returned with status code [200], response: {'plug': {'connectionState': 'Connected'`

`DEBUG:skodaconnect.connection:Got vehicle data {'plug': {'connectionState': 'Connected', `

printout
```
TMBXXXXXXXXX Charging cable connected - (charging_cable_connected)
        str_state: Charging - state: True
        attributes: {}

```

cable is connected but it doesn't charge right now, variable "instrument.str_state" seems to be wrong.

can it be this row?
https://github.com/lendy007/skodaconnect/blob/d86ce730199fdb66ec91c01cfc38b0ce094477ee/skodaconnect/dashboard.py#L149

**Comment by flopp999** [comment]
https://github.com/lendy007/skodaconnect/blob/d86ce730199fdb66ec91c01cfc38b0ce094477ee/skodaconnect/dashboard.py#L969

https://github.com/lendy007/skodaconnect/blob/6525f3d112cb8be52dc190a72fc039de389b7801/skodaconnect/vehicle.py#L1064

**Comment by Farfar** [comment]
Should be solved in 1.0.41
